nss-ivory-trolls/BangazonAPI
# Ac employee controller

# Description
Feature adds CRUD functionality to Employee Controller. Added automated testing to Employee 
Controller. Updated README.md to include ERD, project intro, Employee Controller instructions and Department Controller instructions.

## Type of change

# Testing Instructions
1. git checkout ac-employee-controller
1. dotnet run
1. open PostMan

Ensure the following HTTP requests are functional:
- Get (All)
- Get (Id)
- Post
- Put

1. Navigate to TestBangazonAPI project. Ensure that all tests in EmployeeCRU file pass. 

# Checklist:
